README.md

```diff
@@ -189,6 +189,22 @@ docker run \
   speccy lint openapi.yaml
 ```
 
+### Using with lint-staged
+
+To lint your specifications before committing them you can use [lint-staged](https://github.com/okonet/lint-staged) to run speccy before each commit. Just install lint-staged and husky as `devDependencies` and add the following to your `package.json`:
+
+```
+"husky": {
+  "hooks": {  
+    "pre-commit": "lint-staged"
+  }
+},
+"lint-staged": {
+  "*.{yml, yaml}": ["speccy lint openapi.yml", "git add"]
```

Comment:
I’m not sure how lint staged works, but this seems like it might be aaaall yaml files?